grabbit/src/main/groovy/com/twcable/grabbit/server/batch/steps/jcrnodes/JcrNodesProcessor.groovy

```diff
@@ -24,12 +24,15 @@ import com.twcable.grabbit.proto.NodeProtos.Properties
 import com.twcable.grabbit.proto.NodeProtos.Property
 import groovy.transform.CompileStatic
```

Comment:
There has been an issue with server side logic so currently it has been removed and added a internal jira ticket. I have pasted the link on internal grabbit channel.